### ISSUE TITLE
No binary available on link from readme file

### ISSUE THREAD

**Issue by jonasgreen88** [OPCFoundation/Misc-Tools#12]
Hi,

I am looking for a pre-built binary of the Certificate Generator tool. The link in the readme file works, but there is no binary of this tool available on that web page, only sample applications. I don't succeed to build it myself. I get errors in the cmake process of openssl. Is it possible for you to distribute a binary?

Regards, Jonas.

**Comment by rudoq32** [comment]
Hello Jonas,
it is a part of installation of some OPC UA software. 
I found it in my file system as Opc.Ua.CertificateGenerator.exe in c:\Program Files (x86)\Common Files\OPC Foundation\UA\v1.0\Bin\.
You may try "Developer Tools » Samples and Tools » Classic » Core Components".
BR Rudo

**Comment by jonasgreen88** [comment]
Hi,

Thanks for the comment. I actually already had it installed then.

Best regards, Jonas.

**Comment by davy7125** [comment]
Hello,

I installed the core components here: https://opcfoundation.org/developer-tools/samples-and-tools-classic/core-components/
But there is nothing related to a certificate generator.

The link in the readme is wrong. I downloaded the content of
https://opcfoundation.org/developer-tools/developer-kits-unified-architecture/-net-stack-and-sample-applications/
and there is still nothing related to a certificate generator.

Is it possible to embed an executable here on GitHub (release section)?

Regards,
Davy
